File: servidor/routes/cuentas.go
package routes

import (
	/*"proyectos/proyecto-leo/servidor/middleware"*/
	"proyectos/proyecto-leo/servidor/services"

	"github.com/gin-gonic/gin"
)

func CuentasRoutes(route *gin.Engine) {

	userGroup := route.Group("/cuentas")
	{
		userGroup.GET("/find", services.ListarCuentas)
		userGroup.GET("/modal/:id_usuario/:id_mesa", services.ListarCuentaPorUsuario)
		userGroup.GET("/cuenta/:id_usuario/:id_mesa", services.ConsultarTotalReserva)
		userGroup.GET("/total/:id_usuario/:id_mesa/:id_cuenta", services.ConsultarPrecioPorPlatoReserva)
		userGroup.GET("/find/:id_plato", services.ConsultaParaActualizar)
		userGroup.DELETE("/delete/:id", services.EliminarCuenta)
		userGroup.PUT("/update/:id", services.ActualizarCuenta)
		userGroup.POST("/add", services.AgregarCuenta)

		userGroup.POST("/grafica/:month", services.CuentaTotalPorAno)
		userGroup.GET("/listar/anos", services.ListarAnos)

	}
}

File: servidor-express/src/routes/cuentas.js
const router = require('express').Router()
const cuentasController = require('../controllers/cuentasController')

router.get("/find", cuentasController.ListarCuentas)
router.get("/modal/:id_usuario/:id_mesa", cuentasController.ListarCuentaPorUsuarioYMesa)
router.get("/cuenta/:id_usuario/:id_mesa", cuentasController.ObtenerPrecioTotalCuenta)
router.get("/total/:id_usuario/:id_mesa/:id_cuenta", cuentasController.ObtenerPrecioPorPlato)
router.get("/find/:id_plato", cuentasController.ObtenerDatosDelPlato)
router.delete("/delete/:id", cuentasController.EliminarCuenta)
router.put("/update/:id", cuentasController.ActualizarCuenta)
router.post("/add", cuentasController.AgregarCuenta)
router.post("/grafica/:month", cuentasController.ObtenerPrecioPorAnoPorCadaMes)
router.get("/listar/anos", cuentasController.ListarAnos)


module.exports = router